client/components/NavBar.jsx

```diff
@@ -108,15 +145,42 @@ export default function NavBar() {
             )}
             {loggedIn && (
               <NavItem className="notif-col">
-                <Link href="#notifications">
-                  <a>
-                    <img
-                      className="nav-img"
-                      src="/navbar-images/notification-icon.svg"
-                      alt="Notifications"
-                    />
-                  </a>
-                </Link>
+                {displayNotifDot ? (
+                  <img
+                    className="nav-img"
+                    src="/navbar-images/notification-icon-red.svg"
+                    alt="Notifications"
+                    id="notif-icon"
+                  />
+                ) : (
+                  <img
+                    className="nav-img"
+                    src="/navbar-images/notification-icon.svg"
+                    alt="Notifications"
+                    id="notif-icon"
+                  />
+                )}
```

Comment:
nitpick but the ternary could be shortened to
```jsx
<img
  className="nav-img"
  src={displayNotifDot ? "/navbar-images/notification-icon-red.svg" : "/navbar-images/notification-icon.svg"}
  alt="Notifications"
  id="notif-icon"
/>
```
for clarity on what the conditional is switching between